_posts/2016-10-16-how-i-removed-two-databases-with-one-pull-request.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How I removed two databases in one pull request
+authors: tge
+---
+
+My first project right after school was called KISS which means
+
+*[Keep it simple, stupid](https://en.wikipedia.org/wiki/KISS_principle)*
+
+
+Three years and a few IT projects later, I thought that I got it right and applied the following rules:
+
+* keep your applications small and simple,
+* remove most of its complexity,
+* don't over-engineer,
+* all code and user stories have to be maintained (<abbr title="You Ain't Gonna Need It">YAGNI</abbr>)
```

Comment:
Isn't YAGNI transversal to stuff like _maintenance debt_, _gold plating_ and so on?